spec/unit/crypto/CrossSigningInfo.spec.js

```diff
@@ -79,54 +87,71 @@ describe("CrossSigningInfo.getCrossSigningKey", function() {
         expect(ab).toEqual({a: 106712, b: 106712});
     });
 
-    it("should request a key from the cache callback (if set) and does not call app" +
-         " if one is found", async () => {
-        const getCrossSigningKey = jest.fn().mockRejectedValue(
-            new Error("Regular callback called"),
-        );
+    it.each(types)("should request a key from the cache callback (if set)" +
+                   " and does not call app if one is found" +
+                   `%o`,
```

Comment:
%o?

Reply:
ah, that should probably be %s, and I thought I put brackets around it

Reply:
ah no, it sould be `%o`. It does this.
<img width="1312" alt="Screenshot 2020-03-02 at 09 44 55" src="https://user-images.githubusercontent.com/3935731/75664543-84a64780-5c6a-11ea-9f96-26c2396adb07.png">